openedx/core/lib/block_structure/transformers.py

```diff
@@ -39,7 +40,7 @@ def __init__(self, transformers=None, usage_info=None):
                 Transformer Registry.
         """
         self.usage_info = usage_info
-        self._transformers = []
+        self._transformers = {'optimized': [], 'nonoptimized': []}
```

Comment:
If we go with the duck-typing approach and not subclassing (see my recent comment), then the presence of the filter method will indicate what type the transformer is.  So `supports_filter` would be most accurate.